src/Command/CodeStudio/CodeStudioWizardCommand.php

```diff
@@ -305,6 +307,7 @@ protected function getGitLabToken(string $gitlab_host): string {
       "* Create a token and grant it both <comment>api</comment> and <comment>write repository</comment> scopes",
       "* Copy the token to your clipboard",
       "* Run <comment>glab auth login --hostname=$gitlab_host</comment> and paste the token when prompted",
+      "* Or pass <comment>--gitlab-token</comment> option with <comment>cs:wizard</comment> command. Example: <comment>acli cs:wizard --gitlab-token=<TOKEN></comment>",
```

Comment:
Wouldn't they also need to pass --gitlab-host-name ?

Reply:
If hostname is not available, this will fail even before reaching this method. It will fail in https://github.com/acquia/cli/blob/master/src/Command/CodeStudio/CodeStudioWizardCommand.php#L319 where we first get the hostname